internal/model/service.go

```diff
@@ -4,20 +4,40 @@ import (
 	"context"
 	"fmt"
 	"strings"
+
+	"github.com/mitchellh/hashstructure"
 )
 
 type ServiceName string
 
 func (s ServiceName) String() string { return string(s) }
 
 type Service struct {
-	K8sYaml        string
+	K8sYaml        string `hash:"ignore"`
 	DockerfileText string
 	Mounts         []Mount
 	Steps          []Cmd
 	Entrypoint     Cmd
```

Comment:
hmm I'm not conviiiinced we need to enforce ALL of these being constant to use an image for the same service... off the top of my head i knew we def want dockerfile, we might want entrypoint for incremental build, and idk if we need mounts or steps...

Reply:
I'll adjust this in the next step where we actually make use of this code.